include/auth_ejabberd.php

```diff
@@ -320,7 +324,7 @@ private function writeLog($sMessage) {
 	 * @param string $sMessage The logfile message
 	 */
 	private function writeDebugLog($sMessage) {
-		if ($this->bDebug)
+		if ($this->bDebug) {
```

Comment:
I have now included it in my `pre-commit` hook. Nice work for that. :-)

And we should eliminate `?>` all together as it is discouraged.